README.md

```diff
@@ -31,12 +31,8 @@ Plates is a native PHP template system that's fast, easy to use and easy to exte
 
 Plates is available via Composer:
 
-```json
-{
-    "require": {
-        "league/plates": "3.*"
-    }
-}
+```
+$ composer require league/plates
 ```
```

Comment:
Looks great, @reinink do you want the `$` or not? I just scanned various php league packages and some have the `$` and some don't.

- http://container.thephpleague.com/installation/
- http://commonmark.thephpleague.com/installation/